index.html

```diff
@@ -68,17 +71,17 @@ <h2>Why?</h2>
       </section>
 
 
-      <section>
+      <section class="quote">
         <blockquote>
-          Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
+          "Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt."
         </blockquote>
         <p>
-          Mx. Ipsum
+          --Mx. Ipsum
         </p>
       </section>
 
 
-      <section>
+      <section class="team">
```

Comment:
If you want to achieve flex for the images within team - try this:
.team ul {
 display: flex;
}

-Flex wants to organize all the direction children so when you choose ul, it will select all the li (line items) to orangize.  And waa laa! From there you can wrap it, center it, etc.